deploy/README.md

```diff
@@ -0,0 +1,214 @@
+
+# JupyterHub on Azure Kubernetes
+
+## Prerequisites
+
+ - [azure client](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+ - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+ - [helm](https://helm.sh/docs/intro/install/)
+
+## Setup Kubernetes Cluster
+Follow this documentation to setup a scaling kubernetes cluster in Azure:
+
+[JupyterHub on azure documentation](https://zero-to-jupyterhub.readthedocs.io/en/latest/microsoft/step-zero-azure-autoscale.html)
+
+**Remarks:**
+ - Use Kubernetes version 1.14: https://github.com/jupyterhub/kubespawner/issues/354. Next time check compatability of the kubernetes python api. Move the newer version if compatible.
+ 
+ ## Setting up Helm
+ Follow this documentation to setup Helm. Helm is a package manager for kubernetes. JupyterHub can be installed with this package manger.
+[JupyterHub setting up Helm](https://zero-to-jupyterhub.readthedocs.io/en/latest/setup-jupyterhub/setup-helm.html)
+
+## Setting up JupyterHub
+Follow this documentation to setup a basic JupyterHub installation. Customizations for the PAOTM course will follow in the next sections. The config.yaml in git already contains some settings for the customizations. For this step the basic config.yaml from the documentations can be used.
+
+[Setting up JupyterHub](https://zero-to-jupyterhub.readthedocs.io/en/latest/setup-jupyterhub/setup-jupyterhub.html)
+
+## Create custom container
+For the course we can make a custom container. The container contains the model and notebook for the course.
+First we need a private container registry to store the container. Create a registry on microsoft Azure with the follow commands:
+Resource group should be the same as the kubernetes cluster.
+
+    az acr create --resource-group <resource group> --name <name container registry> --sku Basic
+
+Login the container registry
+
+	az acr login --name <name container registry>
+
+Attach the container registry to the kubernetes cluster
+
+	az aks update -n <cluster-name> -g <resource group> --attach-acr <name container registry>
+
+The dockerfile for the custom container is stored in thi git repo: Dockerfile-jupyterhub. You first need to download FM from the [build server.](https://build.deltares.nl/buildConfiguration/Dimr_DimrCollector) [This build](https://build.deltares.nl/repository/download/Dimr_DimrCollector/1306704:id/dimrset_lnx64_373.tar.gz) was used for the demo.
+
+ - Download FM, unzip the file and store it in the models folder. 
```

Comment:
From where, which version?

Reply:
build.deltares.nl, build 373  in this example